### PR TITLE
[CU-86b2hy974] promoted commandss out of alpha

### PR DESCRIPTION
**Pull request by patmagee**

### CODE REVIEW

**Comment by platform-automation-dnastack** [comment]
Task linked: [CU-86b2hy974 Promote Instrument integration commands out of alpha](https://app.clickup.com/t/86b2hy974)

File: cloudbuild.yaml [hunk 1]
```diff
@@ -61,10 +61,10 @@ steps:
         git config --global --add safe.directory /workspace
 
         # Public release
-        ./scripts/build-package.py
+        #./scripts/build-package.py
```
Comment (mpanik) [inline cloudbuild.yaml:64]: This is weird, does it mean that you need to do subsequent commit to revert publishing as pre-release?

File: dnastack/cli/workbench/runs_commands.py [hunk 2]
```diff
@@ -518,6 +522,21 @@ def write_logs(iterable: Iterable[bytes], writer):
...
                  required=False
              ),
+             ArgumentSpec(
+                 name='no_defaults',
+                 arg_names=['--no-defaults'],
+                 help='If specified, the command will not use any default values for the workflow.',
+                 as_option=True,
+                 default=False,
+                 required=False
+             ),
```
Comment (mpanik) [inline dnastack/cli/workbench/runs_commands.py:532]: We should do the same for transformations, as far as I know this is already supported by API

File: dnastack/cli/workbench/workflows_versions_defaults_commands.py [hunk 3]
```diff
@@ -2,39 +2,22 @@
...
-def alpha_workflows_versions_command_group():
-    """ Interact with workflow versions """
-
-
-alpha_workflows_command_group.add_command(alpha_workflows_versions_command_group)
-
-
 @click.group('defaults')
-def alpha_workflows_versions_defaults_command_group():
+def workflows_versions_defaults_command_group():
```
Comment (mpanik) [inline dnastack/cli/workbench/workflows_versions_defaults_commands.py:16]: if this is workflows_versions command should we nest it under workflows command?

File: dnastack/cli/workbench/workflow_versions_transformations_commands.py [hunk 4]
```diff
@@ -1,45 +1,24 @@
...
-        match = self.FUNCTION_PATTERN.search(content)
-        if match:
-            return match.group(1)
-        return None
-
-
 @click.group('transformations')
-def alpha_transformations_command_group():
-    """ Interact with workflow transformations """
+def workflow_versions_transformations_command_group():
```
Comment (mpanik) [inline dnastack/cli/workbench/workflow_versions_transformations_commands.py:17]: 1. should we nest it under workflows command? 
1. inconsistency `workflow_versions` vs `workflows_versions`

File: cloudbuild.yaml (hunk 1 above)
Comment (patmagee) [reply cloudbuild.yaml:64]: Yes, that was intentional though at least as a stopgap

**Review by mpanik — APPROVED** [review]
LGTM, left few comments, but none of them is a blocker

**Review by mpanik — APPROVED** [review]

**Pull request merged**